### ISSUE TITLE
Handle decisions from Cinder for Ratings

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#21351]
In the [webhook for Cinder](https://github.com/mozilla/addons-server/issues/21194) we need to handle decisions about ratings.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-539)

**Comment by ioanarusiczki** [comment]
@eviljeff 

Same https://github.com/mozilla/addons-server/issues/21353#issuecomment-1819288494

**Comment by ioanarusiczki** [comment]
@eviljeff 

Thanks for the update. 

I tested deleting a developer reply on -dev and I see in admin that if I delete and write a new reply then a new id is created.

Same for ratings. Or when deleting which had a developer reply. 

I've tested with: 
https://addons-internal.dev.mozaws.net/en-US/admin/models/ratings/rating/?addon=628505
and https://addons-internal.dev.mozaws.net/en-US/admin/models/ratings/rating/?addon=620284

On -stage the id of the first dev reply is reused.
